src/App.tsx

```diff
@@ -341,6 +345,19 @@ export default function App() {
     return <EmptyStatePage pageName={'Nurseries'} />;
   };
 
+  const getInventoryView = (): JSX.Element => {
+    if (selectedOrganization && selectedOrgHasInventory()) {
+      return (
+        <Inventory
+          organization={selectedOrganization}
+          hasNurseries={selectedOrgHasNurseries()}
+          hasSpecies={selectedOrgHasSpecies()}
+        />
+      );
+    }
+    return <EmptyStatePage pageName={'Inventory'} />;
```

Comment:
can we move the empty state logic inside the inventory page?
inside the inventory page we have logic to show the onboarding page - which should take precedence over this empty state page.